content/root/glossary.textile

```diff
@@ -0,0 +1,212 @@
+
+See the "queues documentation":/general/queues for further information.
+
+h2(#realtime-sdk). Realtime client library SDK
+
+The realtime client library SDKs are used to add long-lived realtime connections to Ably within applications.
+
+See the "SDK downloads page":https://ably.com/download for further information on SDKs and the available languages.
+
+h2(#rest-sdk). REST client library SDK
+
+The REST client library SDKs provide an API to interact with the Ably REST (HTTP) interface.
+
+See the "SDK downloads page":https://ably.com/download for further information on SDKs and the available languages.
+
+h2(#rewind). Rewind
+
+Rewind is a feature that enables a client to subscribe to a channel from a position earlier than the current point in time. The @rewind@ parameter can specify an earlier position as a time interval or set number of messages. When a client first attaches to a channel using rewind, all messages from that earlier position are replayed to them.
+
+See the "Rewind documentation":/realtime/channels/channel-parameters/rewind for further information.
+
+h2(#sse). SSE
+
+Server Sent Events (SSE) is an HTTP-based protocol for streaming events from a server to a client. In the context of Ably, SSE can be used to establish a realtime subscription to a set of channels.
+
+See the "SSE documentation":/sse for further information on using SSE with Ably.
+
+h2(#stomp). STOMP
+
+Simple (or Streaming) Text Orientated Messaging Protocol (STOMP) is a protocol that can be used to retrieve messages from a queue.
+
+See "Wikipedia":https://en.wikipedia.org/wiki/Streaming_Text_Oriented_Messaging_Protocol for further information.
+
+h2(#subscriber). Subscriber
+
+A subscriber is an entity that establishes a realtime connection to Ably in order to receive messages on one or more channels.
+
+h2(#webhooks). Webhooks
+
+Webhooks are HTTP callbacks that are triggered by a specified event. Ably supports incoming and outbound webhooks. Incoming webhooks take data from an external application or service and publish it as messages on a channel. Outgoing webhooks notify an external application or service when a message or event is received.
+
+See the "incoming webhook documentation":/general/incoming-webhooks or "outgoing webhook documentation":/general/events for further information.
+
+h2(#websocket). WebSockets
+
+WebSockets are a communication protocol. A WebSocket connection is established with Ably when the realtime client library is instanced. This connection is multiplexed, allowing multiple channels to share a single connection and allows realtime messages to be sent between Ably and the client.
+
+See "Wikipedia":https://en.wikipedia.org/wiki/WebSocket for further information.
```

Comment:
We also have a very informative article on our site https://ably.com/topic/websockets that would be nice to link to.